spec/origin_info_spec.rb

```diff
@@ -0,0 +1,146 @@
+describe "computations from /originInfo field" do
+
+  let(:smods_rec) { Stanford::Mods::Record.new }
+
+  let(:mods_origin_info_start_str) { "<mods xmlns=\"#{Mods::MODS_NS}\"><originInfo>" }
```

Comment:
the reverse, actually, but yes:  old tests and new tests.  have any better ideas of how to do this?  I don't want to spend a lot of time on the old tests b/c i think they will become moot as those methods are deprecated/removed.